orttraining/orttraining/python/training/utils/hooks/_zero_offload_subscriber.py

```diff
@@ -22,6 +22,9 @@
 
 from ._subscriber_base import RuntimeStates, SubscriberBase
 
+from typing import Any, List
+from deepspeed.utils import get_caller_func
```

Comment:
avoid import in the global, because deespeed might not be installed, maybe move it to `_ort_compatible_allgather_fn`?